README.md

```diff
@@ -74,6 +74,7 @@ You can see default config values in `config/development.json`, and override the
 * `hotReloding` enables hot reloading of CSS and React
 * `chrome.debug` enable local chrome development
 * `features.sourceTabs` enable editor tabs
+* `logging.firefoxProtocol` log firefox debugger protocol messages
```

Comment:
I think I would have called it `logging.firefoxProxy` since its logging the Firefox protocol mostly because it needs to go through the proxy and when the proxy goes away so would the pref.